### DISCUSSION TITLE
To enable VoNR

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#182] [General]
hi i gone through some tutorial about vonr ,they mention like volte is similar to Vonr 

is possible , give me some suggestion how to enable vonr to test by using  kamailio .

For my understanding IMS function is same both volte and vonr only core network is different.

Please correct me if i am wrong ?

do you have any pipeline to enable VONR in feature?

**Comment by herlesupreeth** [comment]
VoNR requires kamailio IMS to use SBI interface, sadly its not implemented in kamailio IMS. So its not possible to test VoNR with that IMS, with a third-party IMS definitely yes.

**Reply by Sivanesh1992** [reply]
Thankyou so much . do you have any idea which third party open source ims supporting  for VONR

**Reply by herlesupreeth** [reply]
No idea

**Comment by Sivanesh1992** [comment]
thankyou . do you have any pipeline to implement vonr in future

**Reply by herlesupreeth** [reply]
Unfortunately no, since I stopped working on IMS last year and started working on gNB since then

**Reply by Sivanesh1992** [reply]
hi @herlesupreeth  kamailio IMS  support sms over IMS in 5G SA mode?

**Reply by herlesupreeth** [reply]
I dont think so, since it would require IMS registration to be successful
